spec.md

```diff
@@ -128,16 +128,33 @@ communications could lead to interoperability issues with other platforms.
 
 ## Catalog Management
 
-The first endpoint that a platform will interact with on the broker is the service catalog.
-
-The platform marketplace fetches this endpoint from all brokers in order to present an aggregated user-facing catalog.
-
-Warnings for broker authors:
-
-- Be cautious removing services and plans from their catalogs, as platform marketplaces might have provisioned service instances of these plans. Consider your deprecation strategy.
-- Do not change the ids of services and plans. This action is likely to be evaluated by a platform marketplace as a removal of one plan and addition of another. See above warning about removal of plans.
-
-The following sections describe catalog requests and responses in the Service Broker API.
+The first endpoint that a platform will interact with on the broker is the
+service catalog (`/v2/catalog`). This endpoint returns a list of all services
+available from the broker. Platforms query this endpoint from all brokers
+in order to present an aggregated user-facing catalog.
```

Comment:
`Platforms query this endpoint in order to present an aggregated user-facing catalog.` is better IMO